### PR TITLE
7900303: auto updating components need accessbility control

### PR DESCRIPTION
**Pull request by dbessono**
The auto-updating component that can be improved in JTHarness UI in terms of accessibility is test run progress bar.

Such progress bars are visible on the main desktop and in test run monitor window.

We should provide a mechanism for the user to hide test run progress bars.

### Default state

Screenshot 2024-08-30 at 09 47 36

Screenshot 2024-08-30 at 09 40 22

Visually unchanged comparing to what we have now, without the patch.

### Started with `-Djavatest.desktop.testrunprogressmonitor.hidden=true`

Screenshot 2024-08-30 at 09 48 01

Screenshot 2024-08-30 at 09 46 43

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [CODETOOLS-7900303](https://bugs.openjdk.org/browse/CODETOOLS-7900303): auto updating components need accessbility control (**Enhancement** - P3)

### Reviewers
 * [Alexandre Iline](https://openjdk.org/census#shurailine) - **Reviewer** ⚠️ Added manually

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtharness.git pull/82/head:pull/82` \
`$ git checkout pull/82`

Update a local copy of the PR: \
`$ git checkout pull/82` \
`$ git pull https://git.openjdk.org/jtharness.git pull/82/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 82`

View PR using the GUI difftool: \
`$ git pr show -t 82`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtharness/pull/82.diff">https://git.openjdk.org/jtharness/pull/82.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jtharness/pull/82#issuecomment-2320567361)

### CODE REVIEW

File: src/com/sun/javatest/exec/ProgressMonitor.java [hunk 1]
```diff
@@ -373,11 +373,18 @@ private class ProgressSubpanel extends StatusSubpanel {
...
 
+            meter = new ProgressMeter(colors, state);
             tf = uif.createHeading("pm.tests.mtr");
             uif.setAccessibleInfo(tf, "pm.tests.mtr");
-            add(tf, lnc);
-            add(meter = new ProgressMeter(colors, state), rc);
             uif.setAccessibleInfo(meter, "pm.tests.bar");
+
+            boolean showTestRunProgressMonitor =
+                    !Boolean.valueOf(System.getProperty("javatest.desktop.testrunprogressmonitor.hidden"));
```
Comment (shurymury) [inline src/com/sun/javatest/exec/ProgressMonitor.java:382]: AFAICS in the JavaTest code the property names sometimes are stored in string constants and sometimes are hardcoded in the sources. I can not tell which way is preferred, so just making a comment that optionally a string constant can be used for the property name.

**Comment by shurymury** [comment]
LGTM

File: src/com/sun/javatest/exec/ProgressMonitor.java (hunk 1 above)
Comment (dbessono) [reply src/com/sun/javatest/exec/ProgressMonitor.java:382]: Thank you, filed a follow-up improvement https://git.openjdk.org/jtharness/pull/83

**Pull request closed without merging**